MiniSpace.Services.Reactions/src/MiniSpace.Services.Reactions.Application/Commands/Handlers/CreateReactionHandler.cs

```diff
@@ -64,7 +58,7 @@ public async Task HandleAsync(CreateReaction command, CancellationToken cancella
                 throw new InvalidReactionTypeException(command.ReactionType);
             }
             
-            var reaction = Reaction.Create(command.StudentId, command.StudentFullName, reactionType, contentType, command.ContentId);
```

Comment:
The reaction should have a field StudentFullName. It just shouldn't be passed in request body but taken from IAppContext.Identity.Name. However the StudentName field should be removed from Student.cs and StudentDocument as well, because there is not need for storing them